system/Helpers/text_helper.php

```diff
@@ -492,7 +492,7 @@ function word_wrap(string $str, int $charlim = 76): string
 			}
 		}
 
-		// Use PHP's native function to do the initial wordwrap.
+		// Use PHPs native function to do the initial wordwrap.
```

Comment:
The original is correct. "PHP's" is possessive.

Reply:
Made changes